src/main/kotlin/eu/europa/ec/eudi/prex/Types.kt

```diff
@@ -400,7 +265,7 @@ data class PresentationDefinition(
 @Serializable
 data class DescriptorMap(
     @Required val id: InputDescriptorId,
-    @Required val format: ClaimFormat,
+    @Required val format: String,
```

Comment:
Format value class should be used here (format attribute is not any string but a string that can be converted to JsonObject) along with FormatSerializer

Reply:
I don't think so [details](https://identity.foundation/presentation-exchange/spec/v2.0.0/#presentation-submission)

> The descriptor_map object MUST include a format property. The value of this property MUST be a string that matches one of the [Claim Format Designation](https://identity.foundation/presentation-exchange/spec/v2.0.0/#claim-format-designations). This denotes the data format of the [Claim](https://identity.foundation/presentation-exchange/spec/v2.0.0/#term:claim).